### PR TITLE
Lit7142 semon

### PR DESCRIPTION
**Pull request by MarcinKrawczuk**
I don't have any better idea for a title. You can see the whole thing here: https://eap.bl.uk/archive-file/EAP432-1-34

https://github.com/BetaMasaheft/Documentation/issues/2685

### CODE REVIEW

File: new/LIT7142Semon.xml [hunk 1]
```diff
@@ -0,0 +1,67 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LIT7142Semon" xml:lang="en" type="work">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title xml:lang="gez" xml:id="t1">ስምዖን፡ ዘዓምድ፡</title>
+                <title xml:lang="gez" type="normalized"  corresp="#t1">Sǝmʻon za-ʻāmd</title>
```
Comment (eu-genia) [inline new/LIT7142Semon.xml:9]: please note that for ayn and alif we use ʿ (Unicode 02BF) and ʾ (02BE)  (and not ʻ 02BB and ʼ 02BC which are different unicode symbols used for quotation marks)

https://betamasaheft.eu/Guidelines/?id=transliteration-principles

thank you!

File: new/LIT7142Semon.xml [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+        </revisionDesc>
+    </teiHeader>
+    <text>
+        <body>
+            <div type="bibliography">
+                <listRelation>
+                    <relation name="saws:isAttributedToAuthor" active="LIT7142Semon" passive="PRS11392Simeon"/>
+                    <relation name="saws:isAttributedToAuthor" active="LIT7142Semon" passive="PRS6427Macarius"/>
+                </listRelation>
+            </div><!---->
```
Comment (eu-genia) [inline new/LIT7142Semon.xml:64]: copy incipit as provided in the issue? ንዌጥን፡ በረድኤተ፡ እግዚአብሔር፡ አቡሁ፡ ለእግዚእነ፡ ወመድኃኒነ፡ ወአብራሄ፡  etc

Comment (eu-genia) [reply new/LIT7142Semon.xml:64]: I believe the same text is in EMDA177

**Review by eu-genia — CHANGES_REQUESTED** [review]

File: new/LIT7142Semon.xml (hunk 2 above)
Comment (eu-genia) [reply new/LIT7142Semon.xml:64]: oops sorry I now see that you already provided the link to EMDA177 (which is the same as EAP432-1-34, but we go with the HMML shelfmarks, EAP are all doublets https://github.com/BetaMasaheft/Documentation/issues/2523) 

our link is therefore https://betamasaheft.eu/manuscripts/EMDA177/main

**Review by CarstenHoffmannMarburg — APPROVED** [review]

**Comment by MarcinKrawczuk** [comment]
OK, maybe it's better now.

pt., 15 lis 2024 o 09:34 Eugenia ***@***.***> napisał(a):

File: new/LIT7142Semon.xml [hunk 3]
```diff
@@ -0,0 +1,82 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LIT7142Semon" xml:lang="en" type="work">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title xml:lang="gez" xml:id="t1">ስምዖን፡ ዘዓምድ፡</title>
+                <title xml:lang="gez" type="normalized"  corresp="#t1">Sǝmʾon za-ʻāmd</title>
```
Comment (eu-genia) [inline new/LIT7142Semon.xml:9]: Sorry, it seems to me that the alef has been corrected but the ayn still looks like a different Unicode symbol

**Comment by MarcinKrawczuk** [comment]
OK, done, I'll remember not to copy-paste from vHMML!

pt., 15 lis 2024 o 12:51 Eugenia ***@***.***> napisał(a):

**Review by eu-genia — APPROVED** [review]
Thank you!

**Pull request merged**